jcl/src/java.base/share/classes/java/lang/invoke/LambdaForm.java

```diff
@@ -116,7 +116,13 @@ static BasicType basicType(Class<?> cls) {
 	}
 	/*[IF Java18.3]*/
 	enum Kind {
-		PLACEHOLDER;
+		CONVERT,
+		SPREAD,
+		COLLECT,
+		GUARD,
+		GUARD_WITH_CATCH,
+		LOOP,
+		TRY_FINALLY
```

Comment:
If this change was a separate Pull Request I would have merged it. It doesn't need to be mixed in with the unrelated changes to determine the java version.

Reply:
This `j.l.i.LambdaForm.Kind` enum change alone can't fix `Java 18.3` compile/build errors. 
To fix `Java 18.3` compile/build errors, this `j.l.i.LambdaForm.Kind` enum change need to be applied with other part of this pull request. Agreed, this change can be in a separate PR for quick reviewing/merging. Created https://github.com/eclipse/openj9/pull/377.